src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -50,11 +53,22 @@ public static Project[] getSampleProjects() {
         return projectList;
     }
 
+    public static Task[] getSampleTasks() {
+        return new Task[] {new Task(new TaskDeadline("2022-06-01"), new TaskDescription("Finish 2103")),
+            new Task(new TaskDeadline("2022-07-02"), new TaskDescription("Submit homework")),
+            new Task(new TaskDeadline("2022-08-30"), new TaskDescription("Prepare for 2103 exam"))
+        };
+    }
+
```

Comment:
Should shuffle the data here a bit. This set of data does not allow us to test for sort by deadline properly.